web/src/mobile/components/Orders/index.tsx

```diff
@@ -106,7 +111,7 @@ const OrdersComponent: React.FC = () => {
                 currentTabIndex={currentTabIndex}
                 onCurrentTabChange={setCurrentTabIndex}
                 optionalHead={filteredOrders.length ? renderOptionalHead() : null}
-                isMobileDevice={true}
+                isMobileDevice={withDropdownSelect}
```

Comment:
<Orders /> component is a mobile component, there is no case when <Orders /> can be used for not mobile device. I think this prop is useless